app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] Kindly I recommend you to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the **#update_likes_counter** method every time a new like has been saved to database, then make the **update_likes_counter** method private Just like below.

```
after_save :update_likes_counter

  private

  def update_likes_counter
    post.update(likes_counter: post.likes.count)
  end
```